### PR TITLE
[@command] Enhanced @Despawnmob

### PR DESCRIPTION
**Pull request by nesstea**
Not having to enter the ID in is win.

### CODE REVIEW

File: scripts/commands/despawnmob.lua [hunk 1]
```diff
@@ -10,5 +10,14 @@ cmdprops =
 };
 
 function onTrigger(player, mobId)
-    DespawnMob( mobId );
+    local targ = player:fetchTargetsID();
```
Comment (teschnei) [inline scripts/commands/despawnmob.lua]: d45db767f3f843705e90b041ac8cd2d7aa0c5f27

Comment (nesstea) [inline scripts/commands/despawnmob.lua]: uhh? O.o

Comment (xdemolish) [inline scripts/commands/despawnmob.lua]: he means
https://github.com/DarkstarProject/darkstar/commit/888db905d198dfeeb8c35bea999bdb129149df48

**Comment by nesstea** [comment]
Amended commit for you.

**Pull request merged**